.github/workflows/biome.yaml

```
@@ -0,0 +1,82 @@
+            echo "Running Biome on changed files since ${{ github.event.pull_request.base.ref }}..."
+            BIOME_COMMAND="pnpm exec biome check --changed --since=${{ github.event.pull_request.base.ref }}"
+          else
+            echo "Running Biome on all files..."
+            BIOME_COMMAND="pnpm exec biome check"
+          fi
+          
+          BIOME_OPTIONS="--max-diagnostics=none --no-errors-on-unmatched --files-ignore-unknown=true"
+          
+          BIOME_OUTPUT=$($BIOME_COMMAND $BIOME_OPTIONS) || true
+          COMMAND_STATUS=$?
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)